### ISSUE TITLE
[FEATURE REQ] Add OpenTimestamps client guide

### ISSUE THREAD

**Issue by kristapsk** [raspibolt/raspibolt#1043]
### What & why

Since [v0.15.0](https://github.com/raspibolt/raspibolt/pull/1042) LND timestamps the manifest file along with its signature with [OpenTimestamps](https://opentimestamps.org/). Would be good that RaspiBolt users could also check these before installation.

### How

See https://github.com/opentimestamps/opentimestamps-client for OTS client installation info.

**Comment by VajraOfIndra** [comment]
> ### What & why
> 
> Since [v0.15.0](https://github.com/raspibolt/raspibolt/pull/1042) LND timestamps the manifest file along with its signature with [OpenTimestamps](https://opentimestamps.org/). Would be good that RaspiBolt users could also check these before installation.
> ### How
> 
> See https://github.com/opentimestamps/opentimestamps-client for OTS client installation info.

I had a look and it should be easy to add to the instructions.

Just out of curiosity, I tried to find an example of when this time stamp check would actually prevent something bad to run on my node but couldn't really find any use case.. could you ELI5 the main use case of such a timestamp check for node operators? (assuming the binary checksum has been checked and the signature of the checksum file has also been checked)

**Comment by kristapsk** [comment]
> could you ELI5 the main use case of such a timestamp check for node operators? what bad scenario does it protect against?

You can check that signature was made at release date (or before that), not forged later with different binaries (for example, somebody could gain access to private keys used for signing after original release was made).

>  (assuming the binary checksum has been checked and the signature of the checksum file has also been checked)

Yes, this kinda is enough, but extra security checks never hurts. What if all Github accounts with commit access to RaspiBolt master have been compromised?

**Comment by VajraOfIndra** [comment]
> > could you ELI5 the main use case of such a timestamp check for node operators? what bad scenario does it protect against?
> 
> You can check that signature was made at release date (or before that), not forged later with different binaries (for example, somebody could gain access to private keys used for signing after original release was made).
> 
> > (assuming the binary checksum has been checked and the signature of the checksum file has also been checked)
> 
> Yes, this kinda is enough, but extra security checks never hurts. What if all Github accounts with commit access to RaspiBolt master have been compromised?

Ok so an example would be a checksum and signature that works but the timestamps would not meaning that the release has been modified since its release, right?

Anyways, let's add it for sure! And also for Bitcoin Core. Do you want to have a go at the PR?

**Comment by VajraOfIndra** [comment]
Reopening this issue?

**Comment by kristapsk** [comment]
@VajraOfIndra #1087 is not yet merged into master.

**Comment by VajraOfIndra** [comment]
> @VajraOfIndra #1087 is not yet merged into master.

Gee, I thought it was, sorry.

**Comment by kristapsk** [comment]
@VajraOfIndra You planning to reopen that PR?

**Comment by VajraOfIndra** [comment]
> @VajraOfIndra You planning to reopen that PR?

Yes, will reopen a PR
